src/net/native-stack-impl.hh

```diff
@@ -193,6 +201,7 @@ public:
         : _conn(std::move(conn)) {}
     using data_sink_impl::put;
     virtual future<> put(packet p) override {
+        internal::increment_bytes_sent(p.len());
```

Comment:
Better place is the network interface class.

Reply:
You mean to measure the bytes or to store the counter. I have put the counter into a separate class now as most of the classes in the native stack are templates so it's not immediately easy to put static members there.

Reply:
> You mean to measure the bytes or to store the counter?


Both.

>  I have put the counter into a separate class now as most of the classes in the native stack are templates so it's not immediately easy to put static members there.

It's okay to make the counters namespace-scope.